src/config.rs

```diff
@@ -29,11 +30,13 @@ lazy_static! {
 
 const ORGANIZATION: &str = "Mozilla";
 const APP_NAME: &str = "sccache";
-const DIST_APP_NAME: &str = "sccache-dist";
+const DIST_APP_NAME: &str = "sccache-dist-client";
 const TEN_GIGS: u64 = 10 * 1024 * 1024 * 1024;
 
 pub const HIDDEN_FILE_CONFIG_DATA_VAR: &str = "_SCCACHE_TEST_CONFIG";
 
+pub const INSECURE_DIST_CLIENT_TOKEN: &str = "dangerously_insecure_client";
```

Comment:
This doesn't seem to match the value in the docs. Am I misinterpreting?

Reply:
This token is the one used when you specify `DANGEROUSLY_INSECURE` mode in your config file. The exact value is left undocumented in the readme as there's no reason to know it (it's enough for it to be harcoded).